tests/docker-compose.yml

```diff
@@ -60,17 +60,6 @@ services:
     depends_on: *php_depends_on
```

Comment:
Do we still need docker setups? Travis seems to be handling what's left of the drivers everything pretty well.

Reply:
Maybe for local tests...